core/drivers/crypto/caam/include/caam_desc_helper.h

```diff
@@ -23,14 +24,14 @@ void caam_desc_add_word(uint32_t *desc, uint32_t word);
 
 /* Push/Pop descriptor rings queue */
 #ifdef CFG_CAAM_64BIT
-void caam_desc_push(uint64_t *in_entry, paddr_t paddr);
-paddr_t caam_desc_pop(uint64_t *out_entry);
+void caam_desc_push(struct caam_inring_entry *in_entry, paddr_t paddr);
+paddr_t caam_desc_pop(struct caam_outring_entry *out_entry);
 #else
-void caam_desc_push(uint32_t *in_entry, paddr_t paddr);
-paddr_t caam_desc_pop(uint32_t *out_entry);
+void caam_desc_push(struct caam_inring_entry *in_entry, paddr_t paddr);
+paddr_t caam_desc_pop(struct caam_outring_entry *out_entry);
```

Comment:
looks like the same prototypes no?

Reply:
What do you mean?

Reply:
If i'm right you can remove the `#ifdef`/`#else`:
```c
void caam_desc_push(struct caam_inring_entry *in_entry, paddr_t paddr);
paddr_t caam_desc_pop(uint64_t *out_entry);
```

Reply:
Oh, sorry. I though you were talking about the ```git diff```.
Yes indeed, I will remove it